### DISCUSSION TITLE
Mouse dissapears as soon I start writing (MacOS)

### DISCUSSION THREAD

**Discussion by Crinlorite** [LizardByte/community#395] [Sunshine]
Good afternoon,

After having Sunshine in my main Windows computer at home, and using it to access remotely from out of home with help of a VPN and Moonlight in some devices, I wanted to use this software in MacOS and iPad.

I'm having several errors, from Moonlight (MacOS) to Sunshine (MacOS) the mouse dissapears as soon I start writing, also the backspace and the enter keys sometimes send a double enter and a dobule or tiple backspace, so, this is because Sunshine wrong configuration or Moonlight client?

By the way, the mouse comes back if I start to move the mouse around the screen fast.

Regards,
Crinlorite.

**Comment by Wanting-Ranked-Lavender** [comment]
Can confirm issue is still here. v2024.804.21138.

[a comment by an automated account is not shown]
